src/Common/ValueObject/UserAgentHeader.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Common\ValueObject;
+
+use ArrowSphere\PublicApiClient\AbstractClient;
+
+/**
+ * Holds the User-Agent header value.
+ */
+class UserAgentHeader implements ValueObjectInterface
+{
+    private const FORMAT = 'ArrowSphereClient/%s %s PHP/%s';
+
+    /**
+     * @var string
+     */
+    private $value;
+
+    public function __construct()
+    {
+        $this->value = sprintf(
+            self::FORMAT,
+            AbstractClient::CLIENT_VERSION,
```

Comment:
you could use `\Composer\InstalledVersions::getPrettyVersion('vendor/package');`
see [composer doc](https://getcomposer.org/doc/07-runtime.md)

Reply:
Done.